benchmarks/perf_baseline/run.sh

```
@@ -0,0 +1,105 @@
+	else
+		echo "# FLAGS: $flags   (context only, not a valid baseline)"
+	fi
+	echo "##################################################################"
+
+	# shellcheck disable=SC2086
+	# FMT_HEADER_ONLY: otpch.h drags spdlog/fmt in. The benchmark only needs the
+	# XTEA loop, so make fmt header-only rather than linking the engine's deps.
+	XTEA_NEXT="$OUT/bench_xtea.next"
+	XTEA_LOG="$OUT/bench_xtea.build.log"
+	rm -f "$XTEA_NEXT"
+	if ! "${CXX:-g++}" $flags -std=c++23 -DFMT_HEADER_ONLY -I"$ROOT/src" $PROJECT_INCLUDES \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)